oidc-controller/api/core/aries/present_proof_presentation.py

```diff
@@ -0,0 +1,28 @@
+import json, base64
+from typing import Optional, List
+
+from pydantic import BaseModel, Field
+from api.core.aries import PresentProofv10Attachment, ServiceDecorator
+
+
+class PresentationRequestMessage(BaseModel):
+    # https://github.com/hyperledger/aries-rfcs/blob/main/features/0037-present-proof/README.md#presentation
+    id: str = Field(alias="@id")
+    type: str = Field(
+        "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/present-proof/1.0/request-presentation",
+        alias="@type",
+    )
+    request: List[PresentProofv10Attachment] = Field(
+        alias="request_presentations~attach"
+    )
+    comment: str = None
+    service: ServiceDecorator = Field(alias="~service")
```

Comment:
Let's add a `TODO` to switch to OOB protocol (this may not be the best spot though?).

Reply:
added todo to the `send_connectionless_proof_req` method that uses that class.